dashboard/package.json

```diff
@@ -67,6 +67,7 @@
     "redux-devtools-extension": "^2.13.9",
     "redux-thunk": "^2.4.0",
     "remark-gfm": "^3.0.1",
+    "rxjs": "^7.4.0",
```

Comment:
Mmm, why are we adding a new dep with this PR? has the ts code generator changed?

Reply:
It seems to be required when adding a streaming grpc endpoint. The generated typescript client for the streaming RPC appears to require it.